### PR TITLE
Fix sign for localplayer and viewmatrix

### PR DESCRIPTION
**Pull request by xtherk**

### CODE REVIEW

File: offsets/src/analysis/misc.rs [hunk 1]
```diff
@@ -68,8 +68,9 @@ fn local_entity_handle(f: &mut super::Output, bin: PeFile<'_>) {
 fn local_player(f: &mut super::Output, bin: PeFile<'_>) {
 	// The global instance of C_GameMovement contains as its first member a pointer to local player right after its vtable.
 	let mut save = [0; 4];
-	if bin.scanner().finds_code(pat!("8981??0000 488B1D${'}"), &mut save) {
-		let local_player_ptr = save[1];
+	// old: pat!("8981??0000 488B1D${'}")
+	if bin.scanner().finds_code(pat!("488B05u4' 488D0D???? 4488????? 4C89"), &mut save) {
```
Comment (CasualX) [inline offsets/src/analysis/misc.rs:72]: Replace `u4'` with `${'}`. $ is a symbol that follows these RIP relative addresses and {} just temporarily switches, captures the address with ' and returns to let the rest of the pattern match.

Comment (xtherk) [reply offsets/src/analysis/misc.rs]: Thank you very much, I enjoy improving my code through code reviews.

**Comment by CasualX** [comment]
I've deployed the new version to https://casualhacks.net/apexdream/apexdumper.html can you check that it works and uses your sigs?

**Comment by xtherk** [comment]
> I've deployed the new version to https://casualhacks.net/apexdream/apexdumper.html can you check that it works and uses your sigs?

of course it works

image

**Pull request merged**